CreateCardForIssue/ReadMe.md

```diff
@@ -0,0 +1,40 @@
+# CreateCardForIssue
+
+Crate card for issue in selected column. Typically used to add issue on Kanban when it's milestoned.
```

Comment:
typo
```suggestion
Create a card for the issue in the configured column. Typically used to add an issue on Kanban when it's added to a milestone.
```

Reply:
I kept the `milestoned`, as that's how GitHub calls it, see
```
on:
  issues:
    types: ["milestoned"]
```